src/lib/validators.js

```diff
@@ -3,6 +3,10 @@ import * as ob from 'urbit-ob';
 
 import { isValidAddress } from './wallet';
 
+// via: https://emailregex.com/
+const emailRegExp = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
ah, _that's_ why the official W3C regex allows that! I thought it was a bug and went for this more restrictive version. All things being equal, I prefer the more restrictive requirement of a `sub.domain`, since 99% of `m@tld` will be a typo. That's more of a product decision, but we can just call it ourselves, imo. What do you think?

Reply:
that said, we _are_ working with moderately technical users, though, so me@domain might actually happen here

Reply:
yeah, screw it, W3C here I come

Reply:
Eh, none of our users actually own a TLD, and I highly doubt anyone who does isn't used to needing to work around it. I'm fine requiring `@sub.domain`.